spec/features/catalog_view_spec.rb

```diff
@@ -133,6 +133,11 @@
       expect(page).to have_selector(".document-access.review", text: "Restrictions may apply. See Access Note.")
       expect(page).to have_selector("#component-summary .document-access.restricted", text: "Restricted Content")
     end
+    it "lets someone click on the access restrictions badge", js: true do
+      visit "/catalog/AC136_c2889"
+      click_on "Restrictions may apply. See Access Note."
+      expect(page).to have_content("Access & Use")
```

Comment:
I think this is always there, https://findingaids.princeton.edu/catalog/MC247. Is there content we can check that is only there after you click the link, and not before?